src/firefly/js/ui/UploadOptionsDialog.jsx

```diff
@@ -62,7 +62,7 @@ export class UploadOptionsDialog extends PureComponent {
 
     render() {
         const {where, isLoading} = this.state;
-        const {labelWidth, dialogWidth, preloadWsFile=true} = this.props;
+        const {labelWidth, dialogWidth, preloadWsFile=true, style} = this.props;
```

Comment:
Did you test to see what happen if style is `undefined`?
I am guessing a TypeError when it's being spread in line 112.

Reply:
Sorry, I forgot to add it :)